File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n, string control) {
    int answer = 0;
    
    for (int i =0; i < control.length(); i++)
    switch (control[i]) {
        case 'w':
            n++;
            break;
        case 's':
            n--;
            break;
        case 'd':
            n+=10;
            break;
        case 'a':
            n-=10;
            break;
    }
    
    
    return answer = n;
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.py
def solution(n, control):
    answer = 0
    
    for con in control:
        if con == 'w':
            n += 1
        elif con == 's':
            n -= 1
        elif con == 'd':
            n += 10
        elif con == 'a':
            n -= 10
    answer = n
    return answer